frontend/src/components/copr/CoprBuild.tsx

```diff
@@ -79,32 +92,77 @@ export const CoprBuild = () => {
             <Card>
               <CardBody>
                 <Content component="p">
-                  <strong>
-                    You can install the built RPMs by following these steps:
-                  </strong>
+                  <Title headingLevel="h2">Installation instructions</Title>
+                  Please note that the RPMs should be used only in a testing
+                  environment
+                </Content>
+                <Content component="p">
+                  <Title headingLevel="h3">Repository</Title>
+                  <List isPlain>
+                    <ListItem>
+                      <ClipboardCopy
+                        hoverTip="Copy"
+                        clickTip="Copied"
+                        variant="inline-compact"
+                        isCode
+                      >
+                        sudo dnf install -y dnf-plugins-core
```

Comment:
Just a note, on F41+ the plugin is in `dnf5-plugins`, but it should be installed automatically.

Reply:
`dnf -y install 'dnf-command(copr)'` would be safer, though funnily enough `dnf5-plugins` doesn't provide it (but I'm sure they'll fix it eventually).

Reply:
OTOH it seems to work everywhere (where there is `dnf`). @packit/the-packit-team WDYT?

```suggestion
                        sudo dnf install -y 'dnf*-command(copr)'
```

Reply:
I like the suggestion

Reply:
@nforro @lbarcziova added!